src/main/java/jenny/commands/DeleteCommand.java

```diff
@@ -0,0 +1,64 @@
+package jenny.commands;
+
+import java.util.ArrayList;
+
+import jenny.exceptions.JennyException;
+import jenny.storage.Storage;
+import jenny.tasks.Task;
+import jenny.tasks.TaskList;
+import jenny.util.Ui;
```

Comment:
I like how the import statements have a consistent ordering, making it easy to browse